sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/IncrementalExecution.scala

```diff
@@ -64,7 +64,8 @@ class IncrementalExecution(
     val watermarkPropagator: WatermarkPropagator,
     val isFirstBatch: Boolean,
     val currentStateStoreCkptId:
-      MutableMap[Long, Array[Array[String]]] = MutableMap[Long, Array[Array[String]]]())
+      Option[MutableMap[Long, Array[Array[String]]]] =
+    Option(MutableMap[Long, Array[Array[String]]]()))
```

Comment:
Is this variable never None? There is no point to make Option in that case. If we make it Option, perhaps we should do the same as `MicroBatchExecution.currentStateStoreCkptId`

Reply:
+1

Reply:
I don't think these need to be changed